src/index.html

```diff
@@ -4,9 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div
+       class="calendar calendar__month-length--29 calendar__start-day--sat"
+      >
```

Comment:
rename modifiers like 'block-name--modifier-name--modifier-value'
2-nd p in checklist